### PR TITLE
address reviewer comments for first release

### PR DESCRIPTION
**Pull request by atrigila**
Address @mashehu's (thank you!!) comments from first release PR.

- [x] Change version of nf-test
- [x] Use setup-nf-test instead
- [ ] Update date to release date
- [x] Change Figures in tables to Figures+captions
- [x] Add file titles to `Readme.md`: such as ```csv title="samplesheet.csv"
- [x] Replace table with figures for ordered list 
- [x] Order tools alphabetically based on the tool name
- [x] Check assets/chr_rename_del.txt for chrX X
- [x] Move info in `DEVELOPMENT.md` to  `CONTRIBUTING.md`. Remove  `DEVELOPMENT.md`.
- [x] Check `docs/images/Logo.svg`
- [x]  Add a light-gray background to figures, otherwise the black will blend in with the background on the website in dark mode
- [x] Fix typo in `docs/images/metro/metro.py`
- [x] Check if `docs/images/metro/txt2image.md` is required
- [x] Fix typos in `output.md`: "steps".
- [x] Use one line per sentence in markdown.  Makes the diffs a bit nicer to read.
- [x] Explain in `output.md` where are tsv files located.
- [x] Ensure consistency in VCF/vcf writing across `output.md` and `README.md`.
- [x] Remove duplicate in MULTIQC in `output.md`
- [x] Have three samples in `usage.md` instead of 6.
- [x] Check grammar and typos in `usage.md`.
- [x] Remove info described several times in  `usage.md`.
- [x]  Give each of the tools a higher level heading, so that they appear on the ToC on the right, which might make it easier to navigate.
- [x] Add `groovy` in Custom Tool Arguments section
- [x] Remove `modules/local/*/tags.yml`
- [x] Check indentation in `nextflow.config`
- [x] Remove pattern checking in boolean values (`rename_chr` and `compute_freq`)
- [x] Add a pattern checking for a comma separated list (ask for clarification on this requirement)
- [x] Remove pattern in maximal number of individuals per batch for imputation.
- [x] Add 0 in Minimum genotype likelihood probability P(G|R) in validation data.

### CODE REVIEW

**Comment by atrigila** [comment]
I will continue addressing this changes tomorrow

**Comment by atrigila** [comment]
The update to release date will be done once all changes are merged and reviewers approve the final release PR

File: CITATIONS.md [hunk 1]
```diff
@@ -10,33 +10,33 @@
 
 ## Pipeline tools
 
-- [QUILT](https://pubmed.ncbi.nlm.nih.gov/34083788/)
+- [bcftools](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3198575/)
 
-> Davies, R. W., Kucka, M., Su, D., Shi, S., Flanagan, M., Cunniff, C. M., ... & Myers, S. (2021). Rapid genotype imputation from sequence with reference panels. Nature genetics, 53(7), 1104-1111.
+  > Li, H. (2011). A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data. Bioinformatics, 27(21), 2987-2993.
```
Comment (LouisLeNezet) [inline CITATIONS.md:15]: In the Pipeline tools the ">" is preceded by a white space but not in the software part.
Could you choose one ?

File: docs/images/metro/txt2image.md [hunk 2]
```diff
@@ -1,21 +0,0 @@
-# Install desktop app
```
Comment (LouisLeNezet) [inline docs/images/metro/txt2image.md:1]: Why this deletion ?
I personnaly found it useful when updating all the images.

File: docs/images/metro/phaseimpute.drawio.png
Comment (LouisLeNezet) [inline docs/images/metro/phaseimpute.drawio.png:1]: This pipeline is really nice !
However there is some part that are wrongly connected...
Why not use the global svg instead ?

File: README.md [hunk 3]
```diff
@@ -70,18 +85,6 @@ nextflow run nf-core/phaseimpute \
 
 For more details and further functionality, please refer to the [usage documentation](https://nf-co.re/phaseimpute/usage) and the [parameter documentation](https://nf-co.re/phaseimpute/parameters).
 
-## Description of the different steps of the pipeline
```
Comment (LouisLeNezet) [inline README.md:73]: If I follow correctly all this deleted part has been moved to the usage.md ?

File: nextflow_schema.json [hunk 4]
```diff
@@ -38,8 +38,7 @@
                 },
                 "rename_chr": {
                     "type": "boolean",
-                    "description": "Should the panel vcf files be renamed to match the reference genome (e.g. 'chr1' -> '1')",
-                    "pattern": "true|false"
+                    "description": "Should the panel vcf files be renamed to match the reference genome (e.g. 'chr1' -> '1')"
```
Comment (LouisLeNezet) [inline nextflow_schema.json:41]:
```suggestion
                    "description": "Should the panel VCF files be renamed to match the reference genome (e.g. 'chr1' -> '1')"
```

**Review by LouisLeNezet — CHANGES_REQUESTED** [review]

File: docs/images/metro/phaseimpute.drawio.png
Comment (atrigila) [reply docs/images/metro/phaseimpute.drawio.png:1]: I was not able to make changes to the svg in drawio. 

Which parts are wrongly connected? I can improve those.

File: docs/images/metro/txt2image.md (hunk 2 above)
Comment (atrigila) [reply docs/images/metro/txt2image.md:1]: It was a comment by the reviewer.

Comment (LouisLeNezet) [reply docs/images/metro/txt2image.md:1]: Okay I will keep it separately ^^

File: docs/images/metro/phaseimpute.drawio.png
Comment (LouisLeNezet) [reply docs/images/metro/phaseimpute.drawio.png:1]: There are multiple so we might need to do it in a separate PR and discuss it separately ?

Comment (LouisLeNezet) [inline docs/images/metro/phaseimpute.drawio.png:1]: Could you just removed the bcftools mpileup ?
It is used to call the variants from the full depth bam file and I don't think it is in the spec of this figure.
If you want you could add this figure to the draw.io present in the repo 😉

**Review by LouisLeNezet — APPROVED** [review]
The metro map is really nice to understand which tools are used when.
We can improve it in the future if we want.

File: docs/images/metro/phaseimpute.drawio.png
Comment (atrigila) [reply docs/images/metro/phaseimpute.drawio.png:1]: yes, I will do this and update the diagram

**Pull request merged**